tailfs/tailfsimpl/compositedav/stat_cache_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package compositedav
+
+import (
+	"bytes"
+	"testing"
+	"time"
+
+	"tailscale.com/tstest"
+)
+
+func TestStatCache(t *testing.T) {
+	// Make sure we don't leak goroutines
+	tstest.ResourceCheck(t)
+
+	val := []byte("1")
+	ttl := 1 * time.Second
+	c := &StatCache{TTL: ttl}
+
+	// set new stat
+	c.set("file", 1, val)
+	fetched := c.get("file", 1)
+	if !bytes.Equal(fetched, val) {
+		t.Errorf("want %q, got %q", val, fetched)
+	}
+
+	// fetch stat again, should still be cached
+	fetched = c.get("file", 1)
+	if !bytes.Equal(fetched, val) {
+		t.Errorf("want %q, got %q", val, fetched)
```

Comment:
Very small nit: typically [got before want](https://go.dev/wiki/CodeReviewComments#useful-test-failures)

(there are a few of these)

Reply:
Will fix.

n.b. a benefit of using something like QuickTest or Testify is that you get consistently formatted test failures without having to manually adhere to such conventions :)